client/my-sites/purchases/site-level-purchases-error-boundary.tsx

```diff
@@ -7,7 +7,9 @@ import styled from '@emotion/styled';
 
 const debug = debugFactory( 'calypso:site-level-purchases-error-boundary' );
 
-export default class SiteLevelPurchasesErrorBoundary extends React.Component< SiteLevelPurchasesErrorBoundaryProps > {
+export default class SiteLevelPurchasesErrorBoundary extends React.Component<
```

Comment:
Is this change safe? I seem to remember reading somewhere that this would break some things and was fixed in a recent version of Prettier. But maybe I'm misremembering.